guide/inheritance-wallet/onboarding-cosigners.md

```diff
@@ -0,0 +1,156 @@
+      alt: Screen showing a confirmation dialog.
+      caption: The app asks whether she has enabled all signing devices.
+    - file: alice-enable-device/enable-alice-done
+      alt: Home screen showing the wallet with only one task associated.
+      caption: The last thing to do is to download the wallet backup kit.
+---
+
+<!--
+
+Editor's notes
+
+This page covers how Alice imports the savings wallet to her own app and enables her signing device to create and sign transactions.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1
+
+-->
+
+# Cosigner onboarding
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+[So far]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }}), the wallet has been created on Bob's app. The next step is to get Alice set up as a cosigner and to enable her to sign transactions. To do this Alice needs to:
+
+1. Import the wallet configuration to her app.
+2. Enable her BitBox to sign transactions. 
+
+### Wallet onboarding
+The first step is to add the family savings wallet on Alice's app.
+
+#### Bob shares the wallet configuration
+Bob shares the wallet configuration with Alice by tapping on the "onboard cosigner" task. Just like during wallet creation, this displays a QR code that Alice can scan to import the configuration. Alternatively, Bob can share the configuration over a secure channel.
+
+{% include image-gallery.html pages = page.images_bob-export-config %}
+
+The app also shows an option to view the [wallet descriptor](https://bitcoin.design/guide/glossary/#output-script-descriptor). The wallet descriptor is a small piece of text that contains all the information that a wallet application needs to recover a multi-key wallet. We will cover this topic in more detail in the chapter about the wallet backup.
+
+After Alice has successfully set up the wallet on her end, Bob checks off the task. For our application, this is a manual step, because it does not run a server that coordinates activity between the users. This adds a bit more friction to the user experience but minimizes reliance on the wallet provider.
+
+#### Alice sets up the wallet on her app
+
+Alice will scan the QR code from Bob's phone to import the wallet into her app. She reviews the final wallet configuration and hits "add wallet".
+
+{% include image-gallery.html pages = page.images_alice-import-config %}
+
+On the home screen, the app shows the wallet, along with a task list that reminds her to: 
+- Save the wallet backup kit.
+- Enable her BitBox to sign transactions.
+
+### Enabling the signing devices
+
+Bob and Alice's signing devices need to be enabled to sign transactions for the family savings wallet. This is done by importing the wallet configuration to each device.
+
+#### Bob did it during wallet creation
```

Comment:
Since this page is called Cosigner onboarding it feels superfluous to describe Bob's enabling of his signing devices here. If necessary, move to wallet creation page, or just cut.